docs/getting-started-guides/scratch.md

```diff
@@ -0,0 +1,727 @@
+
+### Networking 
+Each node needs to be allocated its own CIDR range for pod networking.
+Call this $NODE_X_POD_CIDR.
+
+A bridge called `cbr0` needs to be created on each node.  The bridge is explained
+further in the [networking documentation](../networking.md).
+
+- Recommended, automatic approach:
+  1. Set `--configure-cbr0=true` option in kubelet init script and restart kubelet service.  Kubelet will configure cbr0 automatically.
+     It will wait to do this until the node controller has set Node.Spec.PodCIDR.  Since you have not setup apiserver and node controller
+     yet, the bridge will not be setup immediately.
+- Alternate, manual approach:
+  1. Set `--configure-cbr0=false` on kubelet and restart.
+  1. Create a bridge
+  - e.g. `brctl addbr cbr0`.
+  1. Set appropriate MTU
+  - `ip link set dev cbr0 mtu 1460`
+  1. Add the clusters network to the bridge (docker will go on other side of bridge).
+  - e.g. `ip addr add $CLUSTER_CIDR dev eth0`
+  1. Turn it on
+  - e.g. `ip link set dev cbr0 up`
+
+If you have turned off docker ip masquerading to allow pods to talk to each
+other, then you may need to do masquerading just for destination IPs outside
+the cluster network.  For example:
+```iptables -w -t nat -A POSTROUTING -o eth0 -j MASQUERADE \! -d ${CONTAINER_SUBNET}```
+will rewrite the source address from
+the PodIP to the Node IP for traffic bound outside the cluster, and kernel
+[connection tracking](http://www.iptables.info/en/connection-state.html#UDPCONNECTIONS)
+will ensure that responses destined to the node still reach
+the pod.
+
+### Other 
+- Enable auto-upgrades for your OS package manager, if desired.
+- Configure log rotation for all node components (e.g. using [logrotate](http://linux.die.net/man/8/logrotate)).
+- Setup liveness-monitoring (e.g. using [monit](http://linux.die.net/man/1/monit)).
+- Setup volume plugin support (optional)
+  - Install any client binaries for optional volume types, such as `glusterfs-client` for GlusterFS
+    volumes.
+
+### Using Configuration Management
+The previous steps all involved "conventional" system administration techniques for setting up
+machines.  You may want to use a Configuration Management system to automate the node configuration
+process.  There are examples of [Saltstack](../salt.md), Ansible, Juju, and CoreOS Cloud Config in the
+various Getting Started Guides.
+
+## Bootstrapping the Cluster
+
+While the basic node services (kubelet, kube-proxy, docker) are typically started and managed using
+traditional system administration/automation approaches, the remaining *master* components of Kubernetes are
+all configured and managed *by Kubernetes*:
+  - their options are specified in a Pod spec (yaml or json) rather than an /etc/init.d file or
+    systemd unit.
+  - they are kept running by Kubernetes rather than by init.
+  - they run in containers, and so tools like `ps` and `top` are less useful than `docker ps` and `kubectl get
```

Comment:
ps and top on the host will still show processes running in containers e.g. try running `ps ax | grep kube-apiserver` on a master